java/alizer-api/src/main/java/com/redhat/devtools/alizer/api/RecognizerBuilder.java

```diff
@@ -10,8 +10,9 @@
  ******************************************************************************/
 package com.redhat.devtools.alizer.api;
 
-public class LanguageRecognizerBuilder {
-    public LanguageRecognizer build() {
+public class RecognizerBuilder {
+    public LanguageRecognizer languageRecognizer() {
```

Comment:
There is a general pattern to have a build method in a builder why this change ?

Reply:
Now there are two different recognizer that can be created, languageRecognizer and ComponentRecognizer that extends the Recognizer class, so i see this class as a Recognizer factory. If i add a build() method which one would be created? I need to ask for some input but i don't see the advantage to have a build with a switch/if-else, no?